.tests/testScript_plotting.m

```diff
@@ -439,4 +442,127 @@ zlabel sqrt(rpm)
         tf = strcmp(s,'%g');
     end
 end
+end
+
+%% external dependencies
+function R = randp(P, varargin)
```

Comment:
The intended function is actually [this `randp`](https://github.com/sky-s/randx/blob/master/randp.m), but really to keep things simple and reduce dependencies, it'd be better to simply replace the one line calling `randp` with something like `r = u.R*(25 + 5*randn([1e4,1]));`.

Reply:
fixed with https://github.com/iAnyKey/physical-units-for-matlab/commit/8a06263f8c420ec818dc407715aca3022dc169b9
Added your randx as a submodule